router.php

```diff
@@ -50,11 +50,11 @@
 (function () {
     $target = getenv('FUNCTION_TARGET', true);
     if (false === $target) {
-        throw new RuntimeException('FUNCTION_TARGET is not set');
+        $target = 'function';
```

Comment:
Two things here: 
1. `function` is a reserved word in PHP, so if you defined a function named `function`, it would throw a fatal error
2. We don't want to default the function name. This is something the user should have to define, and so should be a required field. There is no appropriate default for this.

Reply:
Alright. I'll revert this change.